### DISCUSSION TITLE
part-ii/2010-130/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#178] [Question comments]
Part II 2010 Riemann Surfaces: Paper 1, Section II, G 

https://questions.tripos.org/part-ii/2010-130/

**Comment by silkca** [comment]
For the first part, use a result from example sheets about the numbers of zeroes and poles of an elliptic function.

For the second part, the simplest way is to leave $f$ as a determinant of a matrix and find ways to make the matrix singular by making the columns equal.

For the last part, consider geometrically what it means for the points $\left(1,\wp\left(z_{i}\right), \wp^{\prime}\left(z_{i}\right)\right) \in \mathbb{C}^{3}$ if we set $w=z_1$ and consider the determinant expression for $f(z_2)=0$.
